src/main/java/mate/jdbc/dao/impl/DriverDaoImpl.java

```diff
@@ -0,0 +1,113 @@
+package mate.jdbc.dao.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.model.Driver;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class DriverDaoImpl implements DriverDao {
+    @Override
+    public Driver create(Driver driver) {
+        String createQuery = "INSERT INTO drivers (name, licenseNumber) "
```

Comment:
```suggestion
        String createQuery = "INSERT INTO drivers (name, license_number) "
```
**check it everywhere**